### PR TITLE
Issue 11

### PR DESCRIPTION
**Pull request by fridim**
these commits solve #11

### CODE REVIEW

File: roles/masters/tasks/scripts.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@
        owner: root
        group: root
        mode: 0755
+       force: no
```
Comment (talset) [inline roles/masters/tasks/scripts.yml]: I think we should overide this script in order to provide every update. And if the customer need a custom script, he have to fork on create a custom base on our upstream ?

File: roles/masters/templates/root/bin/install-registry.sh.j2 [hunk 2]
```diff
@@ -1,48 +1,43 @@
...
+#
+# (meaning, if you want to get the last version, just delete it and re-run the playbook.)
 set -e
+set -x
 
-#delete registry to create new one
-#   oc delete dc docker-registry
-#   oc delete svc docker-registry
-#   for i in $(oc get pods --no-headers | awk '/registry/{print $1}'); do oc delete pod $i; done
+delete() {
```
Comment (talset) [inline roles/masters/templates/root/bin/install-registry.sh.j2:7]: need to delete rc too

File: roles/masters/templates/root/bin/install-registry.sh.j2 [hunk 3]
```diff
@@ -1,48 +1,43 @@
...
+}
 
-#
-# Create the registry with local volume
-#
+create_simple_registry() {
+    oadm registry --create --service-account=registry \
+         --config=/etc/{{ masters_etc_config }}/master/admin.kubeconfig \
+         --credentials=/etc/{{ masters_etc_config }}/master/openshift-registry.kubeconfig \
+         --selector="region=infra"
```
Comment (talset) [inline roles/masters/templates/root/bin/install-registry.sh.j2]: region=infra Sould be 

{{ openshift_registry_selector | default('region=infra')}}

File: roles/masters/templates/root/bin/install-registry.sh.j2 (hunk 2 above)
Comment (xlr-8) [inline roles/masters/templates/root/bin/install-registry.sh.j2:7]: +1

File: roles/masters/templates/root/bin/install-registry.sh.j2 (hunk 3 above)
Comment (fridim) [inline roles/masters/templates/root/bin/install-registry.sh.j2]: good point.
but maybe we should also put

```
openshift_registry_selector: region=infra
openshift_router_selector: region=infra
```

in roles/master/defaults/main.yml
Actually even <code>region=primary</code> as the hosts.template does not have the label infra
those openshift_*_selector variables are not used anywhere for now.

**Comment by talset** [comment]
LGTM

**Pull request merged**